.ci/update-github-page.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+set -e
+
+source ./.ci/util.sh
+
+checkForVariable "GITHUB_TOKEN"
+checkForVariable "READ_ONLY_TOKEN"
```

Comment:
only one variable should stay.
keep READ_ONLY_TOKEN.

you can set READ_ONLY_TOKEN before .ci/update-github-page.sh execution.
we need to clearly show that we need readonly token there, it is fine to pass token with any other access, but script is ok with readonly.

Reply:
`READ_ONLY_TOKEN=$GITHUB_TOKEN ./.ci/update-github-page.sh 10.1`

Reply:
brr.... READ_ONLY_TOKEN vs update-github-page.sh is nonsense.
script need RW token.
please use GITHUB_TOKEN in whole script.
remove all usage of READ_ONLY_TOKEN

Reply:
Changes Done